enterprise/server/backends/userdb/userdb.go

```diff
@@ -579,7 +589,32 @@ func (d *UserDB) createUser(ctx context.Context, tx *db.DB, u *tables.User) erro
 	}
 
 	for _, groupID := range groupIDs {
-		err := tx.Exec("INSERT INTO UserGroups (user_user_id, group_group_id, membership_status) VALUES(?, ?, ?)", u.UserID, groupID, int32(grpb.GroupMembershipStatus_MEMBER)).Error
+		err := tx.Exec(`
+			INSERT INTO UserGroups (user_user_id, group_group_id, membership_status, role)
+			VALUES(?, ?, ?, ?)
+			`, u.UserID, groupID, int32(grpb.GroupMembershipStatus_MEMBER), perms.DefaultRole,
+		).Error
+		if err != nil {
+			return err
+		}
+		// Promote from default role to admin if the user is the only one in the
+		// group after joining.
+		numPreExistingUsers := int64(0)
+		err = tx.Model(&tables.UserGroup{}).Where(
```

Comment:
can you just make this a regular-old-query instead of a gorm WHERE query? Eventually we will remove / change gorm and these will be more difficult to migrate

Reply:
Done